### DISCUSSION TITLE
Capturing console output when trying to log RAW file reading

### DISCUSSION THREAD

**Discussion by RaringCoder** [dlemstra/Magick.NET#685] [Help]
Hi,

I have the following code:

```
                    image.Warning += HandleWarning;
                    image.Progress += ImageOnProgress;
                    image.Read(imageBytes, OverrideSettings(file.Extension));
                    image.Warning -= HandleWarning;
                    image.Progress -= ImageOnProgress;
```

To register handlers to capture incoming warnings, this works well for JPG files and I can spot visual corruption. To support RAW files, I override the `format` to specify NEF. This emits a message to the console along the lines of "Unknown file: data corruption at..." when I give it a file with a visual corruption, and it does not emit this message when a file that appears to be fine. 

However, that message does not come through on the `Warning` or `Progress` events and I cannot seem to find anywhere where I can get a hook into stdout.

Can someone point me in the right direction? Thanks.

**Comment by dlemstra** [comment]
Do you have the exact error message so I can try to figure out where this is coming from?
